framework/src/interfaces/Coupleable.C

```diff
@@ -240,10 +240,16 @@ Coupleable::getFEVar(const std::string & var_name, unsigned int comp) const
   return getFieldVar(var_name, comp);
 }
 
+MooseVariableFieldBase *
+Coupleable::getFieldVar(const std::string & var_name, unsigned int comp)
+{
+  return getVarHelper<MooseVariableFieldBase>(var_name, comp);
+}
+
 const MooseVariableFieldBase *
 Coupleable::getFieldVar(const std::string & var_name, unsigned int comp) const
 {
-  return getVarHelper<MooseVariableFieldBase>(var_name, comp);
+  return const_cast<Coupleable *>(this)->getFieldVar(var_name, comp);
```

Comment:
I suppose since this is a one-liner I can just write out the code. But if people modify one (const/non-const) they will have to remember to modify the other